pkg/kn/commands/route/human_readable_flags.go

```diff
@@ -70,13 +71,14 @@ func printRoute(route *servingv1alpha1.Route, options hprinters.PrintOptions) ([
 }
 
 func calculateTraffic(targets []servingv1alpha1.TrafficTarget) string {
-	var traffic string
+	var traffics []string
 	for _, target := range targets {
-		if len(traffic) > 0 {
-			traffic = fmt.Sprintf("%s, %d%% -> %s", traffic, target.Percent, target.RevisionName)
-		} else {
-			traffic = fmt.Sprintf("%d%% -> %s", target.Percent, target.RevisionName)
+		var tag string
+		if target.Tag != "" {
```

Comment:
I wonder if we should explicitly deal with cases where the API returns empty tag? Perhaps have a token like NOTAG or similar?

Reply:
I don't think that is necessary as an empty tag is not distinguishable from having no tag ("" is the empty string value, too, there is no `nil` string in go).

And when there is no tag there is no need to add something.

However, that tag should be added with a space between revision name and opening parentheses.

Reply:
I disagree. But I can live with it.